src/contacts/services/contacts.service.ts

```diff
@@ -8,23 +8,31 @@ import { HttpService } from '@nestjs/axios'
 import { Campaign, PathToVictory } from '@prisma/client'
 import { lastValueFrom } from 'rxjs'
 import { ListContactsDTO } from '../schemas/listContacts.schema'
+import jwt from 'jsonwebtoken'
 
 type CampaignWithPathToVictory = Campaign & {
   pathToVictory?: PathToVictory | null
 }
 
-const { PEOPLE_API_URL } = process.env
+const { PEOPLE_API_URL, PEOPLE_API_S2S_SECRET } = process.env
 
 @Injectable()
 export class ContactsService {
   private readonly logger = new Logger(ContactsService.name)
+  private cachedToken: string | null = null
+  private tokenExpiration: number = 0
 
   constructor(private readonly httpService: HttpService) {
     if (!PEOPLE_API_URL) {
       throw new BadGatewayException(
         'PEOPLE_API_URL environment variable not configured',
       )
     }
+    if (!PEOPLE_API_S2S_SECRET) {
+      throw new BadGatewayException(
+        'PEOPLE_API_S2S_SECRET environment variable not configured',
+      )
+    }
```

Comment:
The pattern we have elsewhere in the api is to have the check for critical env vars on file load, _outside_ of the class, and to just throw a generic `Error` instance.

That way the app is guaranteed to _not_ boot, and therefor won't pass health checks, and then the box won't be brought into rotation by ECS.